README.md

```diff
@@ -20,13 +20,13 @@
 
 ## Table of Contents
 
--   [Table of Contents](#table-of-contents)
--   [Overview](#overview)
--   [Architecture](#architecture)
--   [Deployment](#deployment)
--   [Configuration](#configuration)
--   [Licensing](#licensing)
--   [Contributing](#contributing)
+*   [Table of Contents](#table-of-contents)
```

Comment:
The tables of contents are generated by VSCode
Can we exclude the list symbol change from - to * ?

Reply:
Which plugin are you using to generate the TOC?

Reply:
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
With SlugifyMode set to GitHub

Reply:
Would you be ok with changing the `markdown.extension.toc.unorderedList.marker` setting to `*`? That cause TOC changes and updates to be generated using `*`.

Settings list: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#supported-settings